tests/unit/store/api.spec.js

```diff
@@ -101,7 +103,7 @@ describe('API Vuex Module', () => {
         cloudUrl: process.env.VUE_APP_CLOUD_URL,
         retries: 5,
         serverUrl:
-          localStorage.getItem('server_url') || process.env.VUE_APP_SERVER_URL,
+          localStorage.getItem(SERVER_KEY) || process.env.VUE_APP_SERVER_URL,
```

Comment:
Should we keep this the same as the actual api store state serverUrl and take out the process.env.VUE_APP_SERVER_URL?

Reply:
We're still falling back on the environment variable (line 9 of the api store index) so I don't think we should remove unless there's a testing reason to do so.

Reply:
Ok - that sounds fair.